### ISSUE TITLE
Thoughts on Base32?

### ISSUE THREAD

**Issue by joshxyzhimself** [StableLib/stablelib#39]
Hi, just thought that this lib is perfect for HOTP / TOTP solutions, except for the base32 part (it currently has base64).

There are other solutions like https://github.com/emn178/hi-base32

But do you think it would make sense to also have a Base32 encode/decode function built-in here?

Something like RFC 4648 https://datatracker.ietf.org/doc/html/rfc4648#section-6 that's compatible with apps like Google Authenticator / Microsoft Authenticator / FreeOTP.

Thank you.

**Comment by dchest** [comment]
Agree, base32 would be great to have. Not sure when I'll have time to implement it, though.
